rdr_service/resource/generators/participant.py

```diff
@@ -357,14 +367,24 @@ def _prep_modules(self, p_id, ro_session):
         consent_dt = None
 
         if results:
+            # Track the last module/consent data dictionaries generated, so we can detect and omit replayed responses
+            last_mod_processed = {}
+            last_consent_processed = {}
             for row in results:
+                consent_added = False
                 module_name = self._lookup_code_value(row.codeId, ro_session)
+                module_name = self._lookup_code_value(row.codeId, ro_session)
```

Comment:
Is this duplicate line (extra `module_name = self._lookup_code_value(row.codeId, ro_session)`) intentional?

Reply:
I think it was a bad cut paste from when I copied the code from the dao generator.  Fixing.